pkgs/games/purpur/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchurl, nixosTests, jre_headless }:
+let
+  version = "1.17.1";
+  build = "1428";
+in
+stdenv.mkDerivation {
+  pname = "purpur";
+  version = "${version}r${build}";
+
+  src = fetchurl {
+    url = "https://api.purpurmc.org/v2/purpur/${version}/${build}/download";
+    sha256 = "0sjrj67bgkn7ivab36aifds337ps2jajwk9628ilyhlwm1i4l089";
+  };
+
+  preferLocalBuild = true;
```

Comment:
Why is this done?

Reply:
I set this because both papermc and minecraft-server also have this set.

Looks like it was originally added to minecraft-server in 5d04dcf4f50fc9cdf6581f0dd555acf99ab086a4

Reply:
Ah, alright. Best to stay consistent i suppose.